webscraping.py

```diff
@@ -0,0 +1,104 @@
+import requests
+import pandas as pd
+from bs4 import BeautifulSoup
+
+header = {'User-Agent': "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) "
+                        "Chrome/89.0.4389.128 Safari/537.36",
+          "X-Amzn-Trace-Id": "Root=1-608078e5-5ab47ee82924df8667c3f0e7"}
+base_url = "https://www.amazon.ca/s?k="
+
+
+# Function for searching amazon products given product name
+def get_amazon_search(search_products):
+    url = "https://www.amazon.ca/s?k=" + search_products
+    print(url)
```

Comment:
Make sure you remove print statements before committing your code.

Reply:
Done.